login/src/main/java/com/mito/login/ui/WelcomeScreen.kt

```diff
@@ -36,49 +50,143 @@ import com.mito.common.navigation.NavigationReferences
 import com.mito.common.navigation.NavigationReferences.ProfileReference.getRoute
 import com.mito.core.navigation.Screen
 import com.mito.login.R
+import kotlinx.coroutines.delay
 
 class WelcomeScreen : Screen {
     override val route: String = NavigationReferences.WelcomeReference.getRoute()
 
     @Composable
     override fun Content(navController: NavHostController) {
         val viewModel = hiltViewModel<WelcomeViewModel>()
+
         WelcomeScreen(navController, viewModel)
     }
 }
 
 @Composable
-fun WelcomeScreen(navController: NavHostController, viewModel: WelcomeViewModel) {
-    Card(
-        modifier = Modifier.fillMaxSize(),
-        colors = CardDefaults.cardColors(Color(0x7CC4F0B8)),
-        elevation = CardDefaults.elevatedCardElevation(6.dp),
-        shape = RoundedCornerShape(25.dp),
+fun WelcomeScreen(navController: NavHostController,viewModel: WelcomeViewModel) {
+
+    var isAnimated by rememberSaveable { mutableStateOf(false) }
+    var cardActivated by rememberSaveable { mutableStateOf(false) }
+    var buttonActivated by rememberSaveable { mutableStateOf(false) }
+    var buttonEnabled by rememberSaveable { mutableStateOf(false) }
+
+    LaunchedEffect(key1 = Unit) {
+        delay(1500)
+        isAnimated = true
+        delay(2000)
+        cardActivated = true
+        delay(1500)
+        buttonActivated = true
+        delay(500)
+        buttonEnabled = true
+    }
+
+    Column(
+        Modifier
+            .fillMaxSize()
+            .background(Color.Transparent)
     ) {
-        InitialScreen(navController= navController, modifier = Modifier)
+
+        AnimatedVisibility(
+            visible = isAnimated,
+            enter = slideInVertically(
+                animationSpec =
+                spring(
+                    stiffness = Spring.StiffnessVeryLow,
+                    dampingRatio = Spring.DampingRatioLowBouncy
+                ),
+                initialOffsetY = { fullHight -> -fullHight }
+            )
+        ) {
+            Card(
+                modifier = Modifier
+                    .size(800.dp)
+                    .padding(
+                        top = 60.dp,
+                        start = 25.dp,
+                        bottom = 25.dp,
+                        end = 25.dp
+                    ),
+                colors = CardDefaults.cardColors(Color(0x7CC4F0B8)),
+                elevation = CardDefaults.elevatedCardElevation(6.dp),
+                shape = RoundedCornerShape(25.dp),
```

Comment:
You might remember to use resources. It is not mandatory by our project, but You should get used to doing it.